index.js

```diff
@@ -9,4 +9,10 @@ const server = http.createServer((req, res) => {
 
 server.listen(port, () => {
   console.log(`Server running on http://localhost:${port}/`);
-});
\ No newline at end of file
+});
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

LGTM!



<!-- This is an auto-generated comment by OSS CodeRabbit -->